upup/models/cloudup/resources/addons/coredns.addons.k8s.io/k8s-1.12.yaml.template

```diff
@@ -84,7 +84,7 @@ data:
           ttl 30
         }
         prometheus :9153
-        forward . /etc/resolv.conf {
+        forward . {{ or (join KubeDNS.UpstreamNameservers " ") "/etc/resolv.conf" }} {
```

Comment:
Is this an unrelated change?

Reply:
Okay, missing feature from Coredns that you need in order to not muck with the node-level DNS? This should perhaps be a separate PR.

Reply:
I thought about making this a separate PR, but it was a one liner, related to current work.